### PR TITLE
Task 1

### PR DESCRIPTION
**Pull request by LeonidShatilo**
### Task: 
- Build simple layout using components with React, which are: **header**, **content (main section)** and **footer**
- Create store variable and pass it down to components using props

### Requirements:
- Use **create-react-app** as a boilerplate for your application (this one is optional, you can create your own Webpack config)
- In your project please follow suggested folder structure: (here you can also find your own way on how to make a decent folder structure, this is just my proposal)

**index.js** file should just import the <App /> component and simply call the ReactDOM.render method. Also you need to store a variable here called **store**. This variable should be an object with required fields: **user** (contains **firstName** and **lastName**) and **avatar** (contains **url** and **alt**).

**index.css** (index.module.scss/index.scss) should just contain all the **basic styles** for your app (like rules for html, body).

Each React component should have its own folder with the jsx and relevant stylesheets for example:

- Then you need to find a way on how to pass the store value through an App component to use this store value in a child components (Header, Content). Footer Component can be hardcoded for now

Here is the diagram which explains basic data flow:

### Optional tasks:
- Use SCSS as a preprocessor
- Use CSS modules
- Create an additional index.js file for each component which will just re-export your component. This is always done to make the import shorter (no hints here, try to research it by yourself)

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Header from '../Header/Header';
+import Main from '../Main/Main';
+import Footer from '../Footer/Footer';
+
+export default function App(props) {
```
Comment (broshkabro) [inline src/components/App/App.js:6]: consider using the destructuring assignment syntax:
`App({ userData })`

File: src/components/App/App.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  return (
+    <>
+      <Header userData={props.userData} />
+      <Main userData={props.userData} />
```
Comment (broshkabro) [inline src/components/App/App.js:10]: It is better to pass the object that is really required by the component (user - Header, avatar - Main).

**Review by broshkabro — APPROVED** [review]
Looks nice! Just a couple of comments for your consideration.

File: src/components/App/App.js (hunk 1 above)
Comment (LeonidShatilo) [reply src/components/App/App.js]: You're right. Fixed!

File: src/components/App/App.js (hunk 2 above)
Comment (LeonidShatilo) [reply src/components/App/App.js]: It really is more convenient. Thank you!

**Pull request merged**